storybook/index.html

```diff
@@ -325,6 +325,21 @@ <h1 class="sectionHeading">BAMP</h1>
                       <div class="muralThumbnail--name">I dont Know</div>
                     </div>
                 </section>
+
+                <section class="demo" data-module-name="navModal">
+                    <p class="navModal--p">Only available in Mobile</p>
+                    <div class="navModal">
+                        <h1 class="navModal--h1">x</h1>
+                        <h2 class="navModal--h2">Home <br> Artwork <br> About Us <br> Programs <br> Events <br> Services <br> Shop <br> Get Involved</h2>
```

Comment:
I don't know the best way to do this but I think here we would want to separate all of the nav items into <a> tags so they're clickable? Right now, it's just the design but we want to make sure that when you click on "Home" for example, it'll go to the homepage.

Reply:
agreed that they should be separate.

also for things like this it's better to use CSS `font-size` than the heading tags because they're not true headings.

the close button will need to be a `button` (because it's clickable but not a link), and the other things will need to be links (because they navigate).

putting that all together, I'd do something like this:

```css
<button class="navModal--close">×</button>
<ul class="navModal--linkList">
    <li><a class="navModal--link" href="...">Home</a></li>
    ...
</ul>
```